bld/namelist_files/use_cases/2000_cam6_osloaero.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
+
+<namelist_defaults>
+
+<tracer_cnst_datapath  chem="trop_mam4">atm/cam/ozone                                             </tracer_cnst_datapath>
+<tracer_cnst_file      chem="trop_mam4">tracer_cnst_CAM6chem_2000climo_3D_monthly_c171004.nc      </tracer_cnst_file>
+<tracer_cnst_specifier chem="trop_mam4">'O3','OH','NO3','HO2'                                     </tracer_cnst_specifier>
+
+<tracer_cnst_datapath  chem="ghg_mam4" >atm/cam/tracer_cnst                                       </tracer_cnst_datapath>
+<tracer_cnst_file      chem="ghg_mam4" >tracer_cnst_halons_3D_L70_1849-2015_CMIP6ensAvg_c180927.nc</tracer_cnst_file>
+<tracer_cnst_specifier chem="ghg_mam4" >'O3','OH','NO3','HO2','HALONS'                            </tracer_cnst_specifier>
```

Comment:
I'm confused. Why are these settings for MAM4?

Reply:
Fixed. That was a bug.